e2e-polybft/framework/test-bridge.go

```diff
@@ -392,7 +398,7 @@ func (t *TestBridge) initialStakingOfGenesisValidators(polybftConfig polybft.Pol
 					"--" + polybftsecrets.AccountDirFlag, path.Join(t.clusterConfig.TmpDir, secret),
 					"--amount", polybftConfig.InitialValidatorSet[i].Stake.String(),
 					"--supernet-id", strconv.FormatInt(polybftConfig.SupernetID, 10),
-					"--native-root-token", polybftConfig.Bridge.RootNativeERC20Addr.String(),
+					"--stake-token", polybftConfig.Bridge.StakeTokenAddr.String(),
```

Comment:
We can get it from genesis

Reply:
As per comment: https://github.com/0xPolygon/polygon-edge/pull/1520#discussion_r1200858670